item-api/src/main/java/com/example/auction/item/api/ItemService.java

```diff
@@ -73,19 +72,25 @@
      */
     Topic<ItemEvent> itemEvents();
 
+
+
     @Override
     default Descriptor descriptor() {
         return named("item").withCalls(
                 pathCall("/api/item", this::createItem),
                 restCall(Method.POST, "/api/item/:id/start", this::startAuction),
                 pathCall("/api/item/:id", this::getItem),
+                pathCall("/api/item/:id/update", this::updateItem),
```

Comment:
I don't know why I comited that. 😭 I was in rage trying to fix a serialization issue and removed that.